src/script/components/app-button.ts

```diff
@@ -21,9 +21,12 @@ export class AppButton extends LitElement implements AppButtonElement {
           display: block;
 
           --font-size: var(--desktop-button-font-size);
+          --button-height: 44px;
+          --button-height: var(--desktop-button-height);
```

Comment:
fallbacks as the format `var(--variable, 44px);` doesn't work in the css tag functions, and not really well in the browser.